ctapipe/containers.py

```diff
@@ -216,6 +217,37 @@ class CameraHillasParametersContainer(BaseHillasParametersContainer):
     psi = Field(nan * u.deg, "rotation angle of ellipse", unit=u.deg)
 
 
+class ImageFitParametersContainer(CameraHillasParametersContainer):
```

Comment:
We added support for the toy models in telescope frame now, so it should be possible (and should be the default) that this happens also in telescope frame (i.e. using fov_lon / fov_lat in degrees instead of x, y in m)